### PR TITLE
AVX-86 create AP_Periph 1.0.0 release for ADSB

### PR DESCRIPTION
**Pull request by rabad-matternet**
- updated hwdef.dat to remove dependency on f303-periph
- removed unused features (e.g. SPI, I2C, etc)
- ADSB connected to UART2
- changed version to 1.0.0

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat [hunk 1]
```diff
@@ -1,48 +1,138 @@
...
+define CAN_APP_NODE_NAME "com.matternet.trafficmonitor"
+
+# use DNA
+define HAL_CAN_DEFAULT_NODE_ID 0
+
+# ADSB enable
+define ADSB_PORT hal.uartA
+define HAL_PERIPH_ENABLE_ADSB
+#undef HAL_PERIPH_ADSB_BAUD_DEFAULT
+define HAL_PERIPH_ADSB_BAUD_DEFAULT 57600
```
Comment (zalice) [inline libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat:133]: Why did the default baud decrease from 115200 to 57600?

File: libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat [hunk 2]
```diff
@@ -1,48 +1,138 @@
...
+
+# USART2 for ADSB
+PA2 USART2_TX USART2 SPEED_HIGH
+PA3 USART2_RX USART2 SPEED_HIGH
+
+# USART3 for Debug
+#PB10 USART3_TX USART3 SPEED_HIGH NODMA
+#PB11 USART3_RX USART3 SPEED_HIGH NODMA
+
+# spi bus for compass
```
Comment (AydenChubbic) [inline libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat]: Should "spi" be capitalized?

**Review by AydenChubbic — APPROVED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat (hunk 1 above)
Comment (rabad-matternet) [reply libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat:133]: the pingRX ADS-B is at 57600.  From what I can see in the [datasheet](https://uavionix.com/downloads/pingrx/pingRX-Datasheet.pdf), that baud rate cannot be changed.

File: libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat (hunk 2 above)
Comment (rabad-matternet) [reply libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat]: This actually should be deleted

File: Tools/AP_Periph/version.h [hunk 3]
```diff
@@ -1,12 +1,12 @@
 #pragma once
 
-#define THISFIRMWARE "AP_Periph V1.1"
+#define THISFIRMWARE "AP_Periph V1.0"
```
Comment (bhiln) [inline Tools/AP_Periph/version.h:3]: Are you reverting the version down to 1.0? Was something wrong with 1.1?

**Review by bhiln — APPROVED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat (hunk 1 above)
Comment (zalice) [reply libraries/AP_HAL_ChibiOS/hwdef/f303-ADSB/hwdef.dat:133]: @tridge do you know why HAL_PERIPH_ADSB_BAUD_DEFAULT was set to 115200 before? was it redefined elsewhere downstream to 57600?

**Comment by rabad-matternet** [comment]
Abandoning this pull request (and branch).  Deprecated.

**Pull request closed without merging**